src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else if (days <= 6) {
+    return (days * 40) - 20;
+  }
```

Comment:
Since you return a value in each condition, you can create two separate if instead of an else if statement.

Also move "Magic numbers" like "20", "40" and "50" to separate constant variables